lib/pbio/drv/usb/usb_ev3.c

```diff
@@ -214,11 +216,15 @@ static uint32_t pbdrv_usb_setup_misc_tx_byte;
 // Whether the device is using USB high-speed mode or not
 static bool pbdrv_usb_is_usb_hs;
 
+// Whether there is a host app listening to events
+static bool pbdrv_usb_is_events_subscribed;
+
 // Buffers, used for different logical flows on the data endpoint
 static uint8_t ep1_rx_buf[PYBRICKS_EP_PKT_SZ_HS];
 static uint8_t ep1_tx_response_buf[PYBRICKS_EP_PKT_SZ_HS];
 static uint8_t ep1_tx_status_buf[PYBRICKS_EP_PKT_SZ_HS];
 static uint8_t ep1_tx_stdout_buf[PYBRICKS_EP_PKT_SZ_HS];
+#define USB_RESPONSE_ACTUAL_SZ      (1 + sizeof(uint32_t))
```

Comment:
Isn't this something that should be shared by all implementations? I.e. something defined in protocol.h.

Could even make it more general and make a USB message size wrapper macro to wrap the existing macros.

```c
/**
 * Size of USB messages for Pybricks USB interface.
 *
 * USB has one extra byte header for a message type discriminator
 * compared to BLE messages.
 */
#define PBIO_PYBRICKS_USB_MESSAGE_SIZE(n) (1 + n)
```